src/semver/version.py

```diff
@@ -86,7 +90,15 @@ class Version:
                 (?:\.[0-9a-zA-Z-]+)*
             ))?
             $
-        """,
+        """
+    #: Regex for a semver version
+    _REGEX = re.compile(
+        _REGEX_TEMPLATE.format('', ''),
+        re.VERBOSE,
+    )
+    #: Regex for a semver version that might be shorter
+    _REGEX_OPTIONAL_MINOR_AND_PATCH = re.compile(
+        _REGEX_TEMPLATE.format('?', '?'),
```

Comment:
As we've introduced these placeholders before, we can now use them:

```suggestion
        _REGEX_TEMPLATE.format(opt_patch='', opt_minor=''),
        re.VERBOSE,
    )
    #: Regex for a semver version that might be shorter
    _REGEX_OPTIONAL_MINOR_AND_PATCH = re.compile(
        _REGEX_TEMPLATE.format(opt_patch='?', opt_minor='?'),
```